lib/v2/cmde/index.js

```diff
@@ -5,40 +5,70 @@ const rootURL = 'https://www.cmde.org.cn';
-    });
-    await page.goto(link);
-    await page.waitForSelector('.child-list .list ul');
-    const html = await page.evaluate(() => document.querySelector('.child-list .list ul').innerHTML);
-    const title = await page.evaluate(() => document.title);
-    browser.close();
-
-    const $ = cheerio.load(html);
-    const list = $('li');
-
-    const data = {
-        title,
-        link,
-        item: list
-            .map((_, item) => ({
-                title: $(item).find('a').attr('title'),
-                link: new URL($(item).find('a').attr('href'), link).href,
-                pubDate: parseDate($(item).find('span:last-child').text().replace(/\(|\)/g, '')),
-            }))
-            .get(),
-    };
+    const data = await ctx.cache.tryGet(
+        url,
+        async () => {
+            const page = await browser.newPage();
+            await page.setRequestInterception(true);
+            page.on('request', (request) => {
+                request.resourceType() === 'document' || request.resourceType() === 'script' ? request.continue() : request.abort();
+            });
+            await page.goto(url, {
+                waitUntil: 'domcontentloaded',
+            });
+            await page.waitForSelector('.list');
+            const html = await page.evaluate(() => document.documentElement.innerHTML);
+            await page.close();
+
+            const $ = cheerio.load(html);
+
+            return {
+                title: $('head title').text(),
+                description: $('meta[name=ColumnDescription]').attr('content'),
+                items: $('.list ul li')
+                    .toArray()
+                    .map((item) => {
+                        item = $(item);
+                        return {
+                            title: $(item).find('a').attr('title'),
+                            link: new URL($(item).find('a').attr('href'), url).href,
+                        };
+                    }),
+            };
+        },
+    );
+
+    const items = await Promise.all(
+        data.items.map((item) =>
+            ctx.cache.tryGet(item.link, async () => {
+                const page = await browser.newPage();
+                await page.setRequestInterception(true);
+                page.on('request', (request) => {
+                    request.resourceType() === 'document' || request.resourceType() === 'script' ? request.continue() : request.abort();
+                });
+                await page.goto(item.link, {
+                    waitUntil: 'domcontentloaded',
+                });
+                await page.waitForSelector('.text');
+
+                const html = await page.evaluate(() => document.documentElement.innerHTML);
+                await page.close();
+                const $ = cheerio.load(html);
+                item.description = $('.text').html();
+                item.pubDate = parseDate($('meta[name="PubDate"]').attr('content'));
```

Comment:
```suggestion
                item.pubDate = timezone(parseDate($('meta[name="PubDate"]').attr('content')), +8);
```
docs: https://docs.rsshub.app/joinus/pub-date.html